src/image/converters.js

```diff
@@ -45,18 +39,18 @@ export function viewFigureToModel() {
 			return;
 		}
 
-		// Convert view image to model image.
-		const { modelRange } = conversionApi.convertItem( viewImage, data.modelCursor );
+		// Convert view image to model image and save result as a conversion result.
+		// Image is an object, so we are sure that it won't be split by it's children.
+		data = Object.assign( data, conversionApi.convertItem( viewImage, data.modelCursor ) );
```

Comment:
I'm not sure about this. If `convertItem` will return more properties in the future we may overwrite something important. Also, this way, I do not know what are you setting here.

Reply:
Changed.